extensions/rich_text_components/Math/directives/oppia-noninteractive-math.directive.ts

```diff
@@ -36,8 +36,13 @@ angular.module('oppia').directive('oppiaNoninteractiveMath', [
       controller: ['$attrs', function($attrs) {
         var ctrl = this;
         ctrl.$onInit = function() {
-          ctrl.rawLatex = HtmlEscaperService.escapedJsonToObj(
-            $attrs.rawLatexWithValue);
+          ctrl.mathExpressionsContent = HtmlEscaperService.escapedJsonToObj(
+            $attrs.mathContentWithValue);
+          if (
+            ctrl.mathExpressionsContent !== undefined &&
+            ctrl.mathExpressionsContent.hasOwnProperty('raw_latex')) {
```

Comment:
Can you explain the reason behind this check? [and add this as a code comment.]

Reply:
added a code comment